codeclimate-gofmt.go

```diff
@@ -1,13 +1,13 @@
 package main
 
 import (
+	"fmt"
 	"github.com/codeclimate/cc-engine-go/engine"
-	"sourcegraph.com/sourcegraph/go-diff/diff"
+	"github.com/sourcegraph/go-diff/diff"
```

Comment:
I've changed the import from `sourcegraph.com` to `github.com` due to this error that is encountered when trying to run `go mod tidy` 

```
 sourcegraph.com/sourcegraph/go-diff/diff: sourcegraph.com/sourcegraph/go-diff@v0.6.1: parsing go.mod:
        module declares its path as: github.com/sourcegraph/go-diff
                but was required as: sourcegraph.com/sourcegraph/go-diff
```